processors/config/kalSimpTuple_cfg.py

```diff
@@ -121,48 +139,57 @@
 vtx.parameters["kinkRelCollLcio"] = ''
 vtx.parameters["trkRelCollLcio"] = 'KFTrackDataRelations'
 
-cvtx.parameters["debug"] = 0
-cvtx.parameters["vtxCollLcio"]     = 'TargetConstrainedV0Vertices_KF'
-cvtx.parameters["vtxCollRoot"]     = 'TargetConstrainedV0Vertices_KF'
-cvtx.parameters["partCollRoot"]    = 'ParticlesOnCVertices_KF'
-cvtx.parameters["kinkRelCollLcio"] = ''
-cvtx.parameters["trkRelCollLcio"]  = 'KFTrackDataRelations'
-
-
 vtxgbl.parameters["debug"] = 0
 vtxgbl.parameters["vtxCollLcio"]     = 'UnconstrainedV0Vertices'
 vtxgbl.parameters["vtxCollRoot"]     = 'UnconstrainedV0Vertices'
-vtxgbl.parameters["partCollRoot"]    = 'ParticlesOnUVertices'
+vtxgbl.parameters["partCollRoot"]    = 'ParticlesOnVertices'
```

Comment:
The unconstrained and constrained vtx should have different ROOT collection names.

Reply:
whoops, Im not even sure why I removed that U, maybe thought it was a typo lol. Ill change it back